realm/realm-library/src/objectServer/java/io/realm/ErrorCode.java

```diff
@@ -105,9 +105,9 @@ public int intValue() {
      * <p>
      * FATAL: The session cannot be recovered and needs to be re-created. A likely cause is that the User does not
      * have access to this Realm. Check that the {@link SyncConfiguration} is correct. Any fatal error will cause
-     * the session to be become {@link SessionState#STOPPED}.
+     * the session to become SessionState#STOPPED.
```

Comment:
yes, we don't track(do any logic) regarding the session's state, it's up to the ObjectStore to call the `binder_handler` if it needs an `access_token` or when reporting an error (via `error_handler`) 
I'll update the comment though to no hurt your memories 👻